src/modals/import-account-modal/import-account-seed/import-account-seed.form.tsx

```diff
@@ -22,5 +22,5 @@ export const importAccountSeedValidationSchema: SchemaOf<ImportAccountSeedValues
   seedPhrase: seedPhraseValidation,
   password: string(),
   derivationType: mixed<ImportAccountDerivationEnum>().oneOf(Object.values(ImportAccountDerivationEnum)).required(),
-  derivationPath: string().required()
+  derivationPath: string()
```

Comment:
need's validation for derivation path. 
should start with `m` if provided, valid syntax (take an example from extension)